app/views/articles/_sidebar_nav.html.erb

```diff
@@ -50,7 +50,7 @@
     <header class="p-2 pr-0 flex items-center justify-between">
       <h3 class="crayons-subtitle-3">My Tags</h3>
       <a href="/dashboard/following_tags" class="crayons-btn crayons-btn--icon crayons-btn--ghost-dimmed" aria-label="Customize tag priority" title="Customize tag priority">
-        <%= inline_svg_tag("cog.svg", aria: true, class: "crayons-icon") %>
+        <%= inline_svg_tag("cog.svg", aria: false, class: "crayons-icon") %>
```

Comment:
~~When `aria` attribute is set to `false` is it the same as not having aria attribute at all? Or setting `false` is actually meaningful?~~

I've just checked - when we set `aria` attribute to `false` the output is *not* `aria="false"` in actual markup. It just doesn't show up... In that case, I think better practice would be to remove that attribute entirely instead of changing value to `false`. Unless I'm missing something..?

Reply:
I can remove it! `aria: true` was adding an `img` role to the SVG but it had no `<title>` or label, so it would fail in most accessibility checkers. The wrapping link also has an `aria-label` on it which supersedes the SVG anyway for its accessible name, so that's why I went this route.